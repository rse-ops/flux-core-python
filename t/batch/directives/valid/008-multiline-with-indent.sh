#!/bin/sh
# flux: -N4 --exclusive
# flux: --setattr=user.conf="""
# flux: [config]
# flux:   item = "foo"
# flux: """
foo
