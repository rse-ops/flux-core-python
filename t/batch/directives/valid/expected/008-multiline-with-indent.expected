SETARGS(['-N4', '--exclusive'])
SETARGS(['--setattr=user.conf=[config]\n  item = "foo"\n'])
